mozilla/bedrock
# [bug 1422899] Remove old plugincheck template

## Description
- Removes old `/plugincheck/` page template and assets.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422899

## Testing
- Check I didn't miss anything or remove something that's still needed?
- Check that translations still work ok?
